app/routes/company/components/CompanyDetail.tsx

```diff
@@ -23,224 +28,138 @@ type Props = {
-              <button
-                className={styles.showAllEventsButton}
-                onClick={() =>
-                  this.setState({
-                    viewOld: !this.state.viewOld,
-                  })
-                }
-              >
-                {viewOld ? 'Vis kun kommende' : 'Vis tidligere arrangementer'}
-              </button>
-            </div>
-          </tr>
-          {viewOld && <EventTable events={oldEvents} />}
-        </table>
-        {viewOld && showFetchMoreEvents && (
-          <div
-            style={{
-              display: 'flex',
-              width: '100%',
-              marginTop: '10px',
-              justifyContent: 'center',
-            }}
-          >
-            <Icon
-              name="chevron-down-circle-outline"
-              size={35}
-              onClick={fetchMoreEvents}
-              style={{
-                cursor: 'pointer',
-              }}
-            />
-          </div>
-        )}
-      </div>
-    );
+  if (!company) {
+    return <LoadingIndicator loading={loading} />;
   }
-}
 
-const CompanyDetail = (props: Props) => {
-  const {
-    company,
-    companyEvents,
-    joblistings,
-    fetchMoreEvents,
-    showFetchMoreEvents,
-    loggedIn,
-  } = props;
+  const sortedEvents = companyEvents.sort(
+    (a, b) => moment(b.startTime).unix() - moment(a.startTime).unix()
+  );
+  const upcomingEvents = sortedEvents.filter((event) =>
+    moment().isBefore(moment(event.startTime))
+  );
+  const oldEvents = sortedEvents.filter((event) =>
+    moment().isAfter(moment(event.startTime))
+  );
 
-  if (!company) {
-    return <LoadingIndicator loading />;
-  }
+  const companyInfo = [
+    {
+      text: company.website,
+      icon: 'globe-outline',
+      link: true,
+    },
+    {
+      text: company.address,
+      icon: 'location-outline',
+      link: false,
+    },
+    {
+      text: company.phone,
+      icon: 'call-outline',
+      link: false,
+    },
+    {
+      text: company.companyType,
+      icon: 'briefcase-outline',
+      link: false,
+    },
+  ];
 
-  const joblistingsList =
-    joblistings &&
-    joblistings.map((joblisting) => (
-      <tr key={joblisting.id}>
-        <td>
-          <Link to={`/joblistings/${joblisting.id}`}>{joblisting.title}</Link>
-        </td>
-        <td>{jobType(joblisting.jobType)}</td>
-        <td>
-          <Year joblisting={joblisting} />
-        </td>
-        <td>
-          {
-            // TODO: this doesn't work, because applicationUrl doesn't exist with list-serializer
-            joblisting.applicationUrl && (
-              <a
-                href={joblisting.applicationUrl}
-                target="_blank"
-                rel="noreferrer"
-              >
-                <strong>SØK HER</strong>
-              </a>
-            )
-          }
-        </td>
-      </tr>
-    ));
   return (
-    <Content>
+    <Content
+      banner={company.logo || company?.logo}
```

Comment:
What's going on here? Is this not the same as
```suggestion
      banner={company?.logo}
```

Reply:
Yeah, that's the same! Didn't look too thorough through the old code